crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -691,18 +699,57 @@ module bridge::bridge {
         let ctx = test_scenario::ctx(&mut scenario);
         let chain_id = chain_ids::sui_devnet();
         let mut bridge = new_for_testing(ctx, chain_id);
-        let inner = load_inner_mut(&mut bridge);
 
-        // initially it's unfrozen
-        assert!(!inner.paused, 0);
+        assert!(!load_inner_mut(&mut bridge).paused, 0);
         // unfreeze it, should abort
-        let msg = message::create_emergency_op_message(
-            chain_ids::sui_devnet(),
-            0, // seq num
-            1, // freeze op
+        unfreeze_bridge(&mut bridge);
+
+        destroy(bridge);
+        test_scenario::end(scenario);
+    }
+
+    #[test]
+    #[expected_failure(abort_code = EBridgeUnavailable)]
+    fun test_execute_send_token_frozen() {
+        let mut scenario = test_scenario::begin(@0x0);
+        let ctx = test_scenario::ctx(&mut scenario);
+        let chain_id = chain_ids::sui_devnet();
+        let mut bridge = new_for_testing(ctx, chain_id);
+
+        assert!(!load_inner_mut(&mut bridge).paused, 0);
+        freeze_bridge(&mut bridge);
+
+        let eth_address = b"01234"; // it does not really matter
+        let btc: Coin<BTC> = coin::mint_for_testing<BTC>(1, ctx);
+        send_token(
+            &mut bridge,
+            chain_ids::eth_sepolia(), 
+            eth_address,
+            btc,
+            ctx,
+        );
+
+        destroy(bridge);
+        test_scenario::end(scenario);
+    }
+
+    #[test]
+    #[expected_failure(abort_code = EInvalidBridgeRoute)]
+    fun test_execute_send_token_invalid_route() {
+        let mut scenario = test_scenario::begin(@0x0);
+        let ctx = test_scenario::ctx(&mut scenario);
+        let chain_id = chain_ids::sui_devnet();
```

Comment:
nit: use testnet(), since we are deprecating devnet id

Reply:
fixed